commit 1094e147160112f0725132a43dad9b07a83da65f (refs/changes/31/2331/5)
Author: Tim Kordas <tim.kordas@joyent.com>
Date:   2017-11-20T15:06:08-08:00 (1 year, 11 months ago)
    
    MANTA-3364 'Loadbalancer has no backends' probe is missing tooling to actually run
    Reviewed by: David Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index 558f979..068a673 100644
--- a/package.json
+++ b/package.json
@@ -11,6 +11,7 @@
         "dashdash": "1.4.0",
         "forkexec": "1.1.0",
         "jsprim": "1.4.0",
+        "haproxy-stat" : "0.1.0",
         "node-uuid": "1.4.1",
         "once": "1.3.0",
         "vasync": "2.0.0",
